configs/2021.03.0.ini

```diff
@@ -0,0 +1,21 @@
+[DEFAULT]
+version=2021.02.0
```

Comment:
Version number different from the `.ini` name above?

Reply:
good catch, I think I fixed it in a later commit but will check again

Reply:
nope I did not, will fix 😬

Reply:
6fa58ad